### DISCUSSION TITLE
What are the primary features of MaSh, and what more can we add ??

### DISCUSSION THREAD

**Discussion by BOSS294** [BOSS294/PROJECT-991#13] [Q&A] [answered]
What are the primary features of MaSh, and what more can we add ??

**Comment by jindalpriyanshu101** [comment, marked as answer]
Currently we've planned following features for MaSh:

- Ability to turn on with system
- Stay in system tray with speech recognition
- Respond to keyword calls (similar working as of Google Assistant & Siri)

Here goes the shortlisted future updates:

- System security by integrating facial recognition with ML Test cases
- OCR Capablities
- OpenAI APIs for Gen-AI tasks
